src/desktop/apps/auction/__tests__/routes.jest.js

```diff
@@ -85,6 +85,15 @@ describe("routes", () => {
       await routes.index(req, res, next)
       expect(res.send).toBeCalledWith("<html />")
     })
+
+    // TODO: Fix the test.
+    xit("works even with the Metaphysics module throwing an error", async () => {
```

Comment:
Ah, retrying. I was linked into my local `Reaction`, which had a different version of styled components, leading to some other errors preventing me from running this test locally.